chemprop/v2/models/multi.py

```diff
@@ -3,17 +3,17 @@
 import torch
 from torch import Tensor, nn
 
-from chemprop.v2.data.dataloader import MulticomponentTrainingBatch
-from chemprop.v2.featurizers.molgraph import BatchMolGraph
+from chemprop.v2.data.collate import MulticomponentTrainingBatch, BatchMolGraph
 from chemprop.v2.nn import (
     MulticomponentMessagePassing,
     Aggregation,
-    OutputTransform,
+    # OutputTransform,
```

Comment:
What is `OutputTransform` for? Why do we comment it out here but set it to `None` below?

Reply:
This is just so the PR actually works. This will actually be fixed in your PR #522